DependencyInjection/Configuration.php

```diff
@@ -102,6 +102,14 @@ protected function configureClients(ArrayNodeDefinition $root)
                         ->cannotBeEmpty()
                         ->info('The service id of a factory to use when creating the adapter.')
                     ->end()
+                    ->booleanNode('flexible_client')
+                        ->defaultFalse()
+                        ->info('If true we will wrap this client in a FlexibleHttpClient which emulates async or sync behavior.')
```

Comment:
i would say: 

Set to true to get the client wrapped in a FlexibleHttpClient which emulates...